### DISCUSSION TITLE
Is there a py3gpp implementation for hNRReferenceWaveformGenerator() and hNRDownlinkEVM() of Matlab 5G toolbox

### DISCUSSION THREAD

**Discussion by amulamoottil** [catkira/py3gpp#33] [General]
Hi @catkira ,
Is there a py3gpp implementation for hNRReferenceWaveformGenerator() and hNRDownlinkEVM() of Matlab 5G toolbox

**Comment by vborchsh** [comment]
Definitely no. But you do it by yourself :)

**Comment by catkira** [comment]
We welcome pull requests :)
